regression/goto-analyzer/flow-sensitive-function-call-recursive/main.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+
+int fun(int other)
+{
+  if(other > 0)
+  {
+    int value = fun(other - 1);
+    return value + 1;
+  }
+  else
+  {
+    return 0;
+  }
+}
+
+int main(int argc, char *argv[])
+{
+  int z = fun(0);
+  assert(z == 0); // Unknown as the function has parameter as top
```

Comment:
`// Sucess because flow-sensitivity blocks the branch`

Reply:
Done